bin/generate-prod-config.js

```diff
@@ -0,0 +1,43 @@
+const _    = require("lodash"),
```

Comment:
Hm... there is no need for configuration atm., but I see the point ... I'll move this.